v0.1/core/anatomicalEntity.schema.json

```diff
@@ -3,7 +3,13 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "description": "Definition of an anatomicalEntity",
   "type": "object",
-  "additionalProperties": false,
+  "additionalProperties": false,   
+  "required": [
+    "@type",
+    "@id",
+    "nameOfLocation",
+    "relatedOntology"
```

Comment:
Thinking about it a bit more, I would remove "relatedOntology" from the required fields.
In case of anatomical entities that belong to a atlas-terminology this should be filled.
In case of a user-defined anatomical entity there might not be yet an ontology existing.